src/Microsoft.DotNet.Build.Tasks/PackageFiles/PerfTesting.targets

```diff
@@ -38,7 +34,7 @@
   </Target>
   <!-- Copy over the performance runners to the test directory-->
   <Target Name="CopyXunitExecutionFiles" AfterTargets="CopyTestToTestDirectory" Condition="'$(Performance)' == 'true'">
-
+    <Error Condition="'$(XunitPerfRunnerPackageVersion)' == ''" Text="Xunit performance package version was not specified" />
```

Comment:
I'm fine with the version check here but I also think it is probably worth an Exists check on XunitPerfAnalysisPath.